src/FileIO.jl

```diff
@@ -11,6 +11,7 @@ export DataFormat,
        add_saver,
        filename,
        file_extension,
+       info,
```

Comment:
Why is `info` being exported in this PR?

Reply:
`info` is already listed in the `FileIO` module docstring, so I added a link and the docstring to the documentation site. The export is just to make more sure that it's part of the public API.

Reply:
Other people may have opinions on exporting `info` so it would be good to make that change in a separate PR.

Reply:
That's fair. I'll push a revert soon. Thanks!

Reply:
Reverted in [a33d30d](https://github.com/JuliaIO/FileIO.jl/pull/393/commits/a33d30df84639e3fd312afc0a2e426b9a1e2d732)!